app/views/teams/teams/my_solutions/show.html.haml

```diff
@@ -32,7 +32,7 @@
 
               %p Well done on submitting your iteration. Your team can now post comments giving feedback, suggestions and ideas.
 
-          =form_for [:teams, @team, DiscussionPost.new], remote: true, html: {class: "new-discussion-post-form"} do |f|
+          =form_for [:teams, @team, DiscussionPost.new], remote: true do |f|
```

Comment:
Since Rails 5.something `remote: true` is default, so you can leave it out. Unless it goes only for `form_with`? 
It's on more places, I only commented it here.

Reply:
Really?! That sounds like it will break everything! Got a reference for me?

Reply:
Hmmm, it's the default for `form_with`, not `form_for` 
"By default form_with attaches the data-remote attribute submitting the form via an XMLHTTPRequest in the background if an Unobtrusive JavaScript driver, like rails-ujs, is used. See the :local option for more."
https://api.rubyonrails.org/v5.1/classes/ActionView/Helpers/FormHelper.html#method-i-form_with

Reply:
So `form_for` is as it was but is soft-deprecated in favour of `form_with` moving forward: https://m.patrikonrails.com/rails-5-1s-form-with-vs-old-form-helpers-3a5f72a8c78a

Reply:
Yep! I started using `form_with` whenever I create a new form, but I don't update existing `form_for`'s.Yet.